src/App/Router.php

```diff
@@ -202,6 +202,7 @@ public function collectRoutes()
 			$collector->addGroup('/2fa', function (RouteCollector $collector) {
 				$collector->addRoute(['GET', 'POST'], '[/]'                    , Module\Settings\TwoFactor\Index::class);
 				$collector->addRoute(['GET', 'POST'], '/recovery'              , Module\Settings\TwoFactor\Recovery::class);
+				$collector->addRoute(['GET', 'POST'], '/app_specific'          , Module\Settings\TwoFactor\AppSpecific::class);
```

Comment:
Wouldn't a route '2fa/app_specific' be more precise? Just asking :)

Reply:
The full route is `settings/2fa/app_specific`.

Reply:
Arg.. Didn't got that point on my mobile view.. Never mind..